apps/backoffice/src/app/api/utils/io-services-cms-proxy.ts

```diff
@@ -37,31 +38,26 @@ export const forwardIoServicesCmsRequest = (
 ) => async <T extends keyof IoServicesCmsClient>(
   operationId: T,
   nextRequest: NextRequest,
+  backofficeUser: BackOfficeUser,
   pathParams?: PathParameters
 ) => {
+  console.log("forwardIoServicesCmsRequest");
   // extract jsonBody
   const jsonBody = nextRequest.bodyUsed && (await nextRequest.json());
 
-  // TODO: extract JWT token
-
+  const headerParameters = backofficeUser.parameters;
   // create the request payload
   const requestPayload = {
     ...pathParams,
     body: jsonBody,
-    "x-user-email": "SET_RETRIEVED_USER_EMAIL_HERE", // TODO: replace with real value
-    // "x-user-groups": "SET_RETRIEVED_USER_GROUPS_HERE", // TODO: replace with real value
-    "x-user-id": "SET_RETRIEVED_USER_ID_HERE", // TODO: replace with real value
-    "x-subscription-id": "SET_RETRIEVED_SUBSCRIPTION_ID_HERE", // TODO: replace with real value
-    "x-user-groups": "SET_RETRIEVED_USER_GROUPS_HERE" // TODO: replace with real value
+    "x-user-email": headerParameters.userEmail,
+    "x-user-id": headerParameters.userId,
+    "x-subscription-id": headerParameters.subscriptionId,
+    "x-user-groups": headerParameters.userGroups.toString()
```

Comment:
**chore:** use join to "comma-separate" values
```suggestion
    "x-user-groups": headerParameters.userGroups.join(",")
```

Reply:
cffb580ea368f32b910eb6695aff4dad46da7846